gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -132,6 +133,19 @@ class BaselineIdea extends AbstractBaselinePlugin {
     }
 
     /**
+     * Configure the default working directory of JUnit tests to be the module directory.
+     */
+    private void addJUnitWorkingDirectory(node) {
+        node.append(new XmlParser().parseText('''
+            <component name="RunManager">
+                <configuration default="true" type="JUnit" factoryName="JUnit">
```

Comment:
Good call -- if we do it for debug, would make sense for the Application configuration as well. Will add that.